packages/account/src/Sections/Profile/PhoneVerification/phone-verification-card.tsx

```diff
@@ -1,14 +1,34 @@
+import { Tag } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv-com/translations';
+import { usePhoneNumberVerificationSessionTimer } from '@deriv/hooks';
 import clsx from 'clsx';
 import React from 'react';
 
 type TPhoneVerificationCard = {
     is_small_card?: boolean;
 };
 
-const PhoneVerificationCard = ({ children, is_small_card }: React.PropsWithChildren<TPhoneVerificationCard>) => (
-    <div className={clsx('phone-verification__card', { 'phone-verification__card--small-card': is_small_card })}>
-        {children}
-    </div>
-);
+const PhoneVerificationCard = ({ children, is_small_card }: React.PropsWithChildren<TPhoneVerificationCard>) => {
+    const { formatted_time } = usePhoneNumberVerificationSessionTimer();
+    return (
+        <div className={clsx('phone-verification__card', { 'phone-verification__card--small-card': is_small_card })}>
+            <div className='phone-verification__card--session-timeout-component'>
+                <Tag
+                    showIcon={false}
+                    size='sm'
+                    variant='custom'
+                    color='rgba(0, 0, 0, 0.04)'
```

Comment:
Should we a color to handle both dark and light themes?

Reply:
updated the code to use hex instead, which will look more visible in dark theme 🙏 

![Screenshot 2024-09-23 at 1 14 12 PM](https://github.com/user-attachments/assets/b470ca90-eb56-4c5c-8661-08053341ef3e)